packages/themes/src/themes/console/dark.ts

```diff
@@ -87,6 +105,17 @@ export const darkTheme = {
       borderWeak: '#4d5266',
       borderWeakDisabled: '#4d5266',
       borderWeakHover: '#4d5266',
+      icon: '#e0e5f0',
+      iconDisabled: '#898fa2',
+      iconStrong: '#ffffff',
+      iconStrongDisabled: '#898fa2',
+      iconStrongHover: '#ffffff',
+      iconStronger: '#151a2d',
+      iconStrongerDisabled: '#898fa2',
+      iconWeak: '#c2cadb',
+      iconWeakDisabled: '#898fa2',
+      iconWeakHover: '#e0e5f0',
+      icontHover: '#e0e5f0',
```

Comment:
There is an issue with the naming of 2 tokens keys `icontHover` instead of `iconHover`